rocks.kfs.FundraisingParticipantSummary/Jobs/FundraisingParticipantSummary.cs

```diff
@@ -185,44 +246,100 @@ public virtual void Execute( IJobExecutionContext context )
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Finish adding group member as recipient.", enableLogging );
 
                             var emailMessage = new RockEmailMessage( systemCommunicationGuid );
+
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Start set recipients.", enableLogging );
                             emailMessage.SetRecipients( recipients );
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Finish set recipients.", enableLogging );
                             var errors = new List<string>();
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Start email send.", enableLogging );
                             emailMessage.Send( out errors );
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Finish email send.", enableLogging );
 
                             if ( errors.Any() )
                             {
-                                errorCount += errors.Count;
-                                errorMessages.AddRange( errors );
+                                errors.ForEach( e => { emailSendErrorMessages.Add( $"{e} - {groupMember.Person.FullName}" ); } );
                             }
                             else
                             {
                                 groupMemberEmails++;
                             }
+                            LogEvent( null, "ProcessCommunication", string.Format( "GroupMember: {0}, Group: {1}", groupMember.Id, groupMember.Group.Id ), "Finish processing group member for communication.", enableLogging );
                         }
                     }
                     else if ( !disablePublicContributionRequests )
                     {
-                        errorCount += 1;
-                        errorMessages.Add( string.Format( "No email specified for {0}.", groupMember.Person.FullName ) );
+                        generalErrorMessages.Add( string.Format( "No email specified for {0}.", groupMember.Person.FullName ) );
                     }
+                    LogEvent( null, "GroupMemberInfo", string.Format( "GroupMember: {0}", groupMember.Id ), "Finish processing group member.", enableLogging );
                 }
             }
-            context.Result = string.Format( "{0} emails sent", groupMemberEmails );
-            if ( errorMessages.Any() )
+            context.Result = $"{groupMemberEmails} Email{(groupMemberEmails == 1 ? string.Empty:"s")} sent.";
```

Comment:
There are helper methods if you want to use them for pluralizing.
```suggestion
            context.Result = $"{groupMemberEmails} {"Email".PluralizeIf( groupMemberEmails > 1 )} sent.";
```